src/commands/config/versions.ts

```diff
@@ -0,0 +1,56 @@
+import { Flags } from '@oclif/core';
+import Command from '../../base';
+
+export default class Versions extends Command {
+  static description = 'Show versions of AsyncAPI tools used';
+
+  static flags = {
+    help: Flags.help({ char: 'h' }),
+  };
+
+  private dependencies: string[] = [];
+  private dependency = '';
```

Comment:
just curious why we need those outside `run` if they are anyway private and needed only inside the scope of the run?

Reply:
Yep. Got carried away by OOP. Changed.